content/posts/abhisheZade_finalreport_Gsoc21.md

```diff
@@ -0,0 +1,80 @@
+After Double tap Pinchzoom is also implemented.There is a small bug in pinch Zoom function which will resolve post GSoC.
+
+Phase 2 started with redesigning UI because there is a huge difference between desktop screen Size and mobile screen size.
+- Designed new menu button.
+- Redesigned Panel for Circuit ELements, Module property , Time Diagram , quick menu.
+- Add new quick tools button. 
+- Designed new nav menu.
+- Implemented UI for responsive version. 
+- Implement lock for Portrait mode.
+
+## FINAL RESULT
+![New Designed UI](/images/abhishek_phase_1_blog/FinalDesignedCVAbhishekZade.png) 
+![New Designed Navmenu](/images/abhishek_phase_1_blog/NamvmenuAbhishekZade.png)
+![Potraid Lock](/images/abhishek_phase_1_blog/potraitlockAbhishekZade.png) 
+
+### Video Link: [click here](https://drive.google.com/file/d/1_B_iZPF_WsSO_0dIdOj3CZI-pWvTiMwi/view?usp=sharing)  
+
+## CHANGES NOT COMPLETED 
+- PinchtoZoom has small bug
+
+## FUTURE CHANGES TO BE DONE
+- Once the simulator is optimized I want to introduce portrait mode since some people are comfortable with portrait mode.
+- Once touch compatibility  gets complete  approval I will integrate in the flutter app using “in app web view”.
+
+## BLOG LINKS
+- [GSoC 2021 My Selection Journey and Community Bonding period @CircuitVerse](https://abhishekzade.medium.com/gsoc-2021-my-selection-journey-and-community-bonding-period-circuitverse-55633d337bde)
+- [GSoC 2021 Phase 1 Blog @CircuitVerse](https://abhishekzade.medium.com/gsoc-2021-phase-1-blog-circuitverse-d09c4e7dfca0)
+- [Touch Compatibility: GSoC 2021 Phase 1 Report](https://blog.circuitverse.org/posts/abhishekzade_phase_1_report/)
+- [Let's Talk About UI](https://abhishekzade.medium.com/lets-talk-about-ui-df0e367f6b4?source=your_stories_page-------------------------------------)
+- [Working of circuitVerse UI Phase 2](https://abhishekzade.medium.com/working-of-ui-f94e9fa07c9a?source=your_stories_page-------------------------------------)
+- [The mistake I have done during the GSoC](https://abhishekzade.medium.com/the-mistake-i-have-done-during-the-gsoc-b7851d70613d?source=your_stories_page-------------------------------------)
+
+## SUMMARY
+In this summer i have successfully implemented the touch compatibility to Circuitverse website with redesigned UI for small screen version.I have  learned so many things this year most important i have meet lots of people had good lifetime bonding.And finally I would like to thanks 
```

Comment:
importantly I have met lots of people and made good lifetime bonding with them. Finally, I would like to thank